### ISSUE TITLE
kafka.yaml.erb - replication stats have wrong domain

### ISSUE THREAD

**Issue by gzysk8** [DataDog/chef-datadog#275]
Metrics `kafka.replication.leader_elections` and `kafka.replication.unclean_leader_elections` are listed under the `kafka.server` domain but should be `kafka.controller` per https://github.com/DataDog/dd-agent/blob/master/conf.d/kafka.yaml.example

Using the default config as is results in:

```
Not Matching: Bean name: "kafka.controller":type="ControllerStats",name="UncleanLeaderElectionsPerSec" - Attribute name: MeanRate - Attribute type: double 
```

**Comment by rlgomes** [comment]
Seems like something that got updated [here](https://github.com/DataDog/dd-agent/blob/master/conf.d/kafka.yaml.example) but the template files in the recipe weren't updated. I'm going to bring this up with their support to see if they can keep these two in sync.

**Comment by olivielpeau** [comment]
Same issue as what #257 addresses, will keep this issue open until we fix it

**Comment by olivielpeau** [comment]
Resolved by #340
